grpc-client-spring-boot-autoconfigure/src/main/java/net/devh/boot/grpc/client/inject/GrpcClientBeanPostProcessor.java

```
@@ -134,6 +141,17 @@ private void processMethods(final Class<?> clazz, final Object bean) {
         }
     }
 
+    /**
+     * Triggers registering grpc client beans from GrpcClientConstructorInjects in bean factory.
+     */
+    private void initGrpClientConstructorInjects() {
+        try {
+            for (GrpcClientConstructorInjection.GrpcClientBeanInjection injection : getConfigurableBeanFactory().getBean(GrpcClientConstructorInjection.class)) {
+                registerGrpcClientBean(injection, injection.getInjectClazz());
+            }
+        } catch (Exception ignored) {
+        }
```

Why would we ignore exceptions here and which?

---

Mainly I want to ignore the `NoSuchBeanDefinitionException` from `getBean` then return. Yep, wrap the whole loop in try block is wrong.